.github/BOTMETA.yml

```
@@ -545,7 +545,7 @@ files:
   $modules/packaging/language/composer.py:
     maintainers: dmtrs resmo
   $modules/packaging/language/cpanm.py:
-    maintainers: fcuny
+    maintainers: russoz
```

Always append, if there isn't the explicit confirmation for a removal (otherwise we have to keep this PR open for a longer time to give that person a chance to react).
```suggestion
    maintainers: fcuny russoz
```